test-harness/tests/open/test_master.py

```diff
@@ -33,7 +33,7 @@ def test_if_valid_user_is_permitted_access(self,
 
 class TestOpenSystemLoggingAgentEndpoint():
     @pytest.mark.parametrize("path", [
-        (""),
```

Comment:
This test here is for a reason - it checks if URLs without path component are handled correctly. Please do not remove it.